compiler/testData/debug/localVariables/suspend/underscoreNames.kt

```diff
@@ -17,36 +17,50 @@ suspend fun box() = foo(A()) { (x_param, _, y_param) ->
 // However, in non-IR, the fields are first stored in local variables, and they are not read directly (even for destructuring components).
 // In IR, the fields are directly read from.
 
-// METHOD : UnderscoreNamesKt$test$2.invokeSuspend(Ljava/lang/Object;)Ljava/lang/Object;
+// The local variable for destructuring lambda arguments, in this case
+// `$dstr$x_param$_u24__u24$y_param`, is moved to a field in the IR backend,
+// so does not figure in the LVT.
 
-// JVM_TEMPLATES
-// VARIABLE : NAME=x_param TYPE=Ljava/lang/String; INDEX=3
-// VARIABLE : NAME=y_param TYPE=Ljava/lang/String; INDEX=4
-// VARIABLE : NAME=this TYPE=LUnderscoreNamesKt$test$2; INDEX=0
-// VARIABLE : NAME=$result TYPE=Ljava/lang/Object; INDEX=1
-
-// JVM_IR_TEMPLATES
-// VARIABLE : NAME=x_param TYPE=Ljava/lang/String; INDEX=2
-// VARIABLE : NAME=y_param TYPE=Ljava/lang/String; INDEX=3
-// VARIABLE : NAME=this TYPE=LUnderscoreNamesKt$test$2; INDEX=0
-// VARIABLE : NAME=$result TYPE=Ljava/lang/Object; INDEX=1
-
-// IGNORE_BACKEND: JVM_IR
 // LOCAL VARIABLES
 // test.kt:12 box: $completion:kotlin.coroutines.Continuation=helpers.ResultContinuation
 // test.kt:4 <init>:
 // test.kt:12 box: $completion:kotlin.coroutines.Continuation=helpers.ResultContinuation
 // test.kt:10 foo: a:A=A, block:kotlin.jvm.functions.Function2=TestKt$box$2, $completion:kotlin.coroutines.Continuation=helpers.ResultContinuation
 // CoroutineUtil.kt:28 getContext:
+
+// LOCAL VARIABLES JVM
 // test.kt:-1 <init>:
 // test.kt:-1 create: value:java.lang.Object=A, completion:kotlin.coroutines.Continuation=helpers.ResultContinuation
 // test.kt:-1 invoke:
+
+// LOCAL VARIABLES JVM_IR
+// test.kt:-1 <init>: $completion:kotlin.coroutines.Continuation=helpers.ResultContinuation
```

Comment:
Instead of rebasing this, we should probably remove (or have an empty) local variable table for synthetic methods with no line numbers?

Reply:
@max-kammerer We would like your thoughts on this, too.

Reply:
Sorry for delay. It's a good question. In old backend compiler doesn't write captured variables to list of local variables.
It would be same difference in next case:
```
fun main() {
    val z = 1;
    {
        z
    }()
}
```
But with old backend debugger able to show captrured variable without this information. So I'm not sure would this difference break smth in debugger experience but keeping old approach looks safer //cc @udalov

Reply:
But from other side:
```
fun main() {
    val z = 1;
        object {
            //JVM: $captured_local_variable$0=1
            //JVM_IR: $z=1
            val x = z;

            fun test() = 1
        }.test()
}
```

Reply:
Looks like lvt in lambda constructor is redundant, but make sense in object literals and local classes

Reply:
Excellent. I think with some additional coverage we can justify leaving out certain LVTs. Are you happy with this PR as is, or should we do more work here?

Reply:
Yes, some tuning could be done in furher prs